### PR TITLE
Exponse parseCommandLine

### LINKED ISSUE #2620: v1.5.0-alpha hides parseCommandLine
To give users of my LS API client/server the same commandline interface they are used to from tsc, I have been using `parseCommandLine`, but quite recently, and sadly affecting `v1.5.0-alpha`, this has been marked `@internal` and is no longer available.

Could you please re-expose this? Ironically, as far as I can tell, the culprit seems to have been a patch that was meant to expose more functions:
```
commit 18d8fedbf52a02af0700f653f499a2e79fb479d3
Author: Mohamed Hegazy <mhegazy@microsoft.com>
Date:   Tue Mar 31 10:52:21 2015 -0700

    Expose readConfigFile and parseConfigFile
```

My users are getting impatient with me depending on the github version of TS, and now I can't offer them use the new npm alpha:-(

### PR DESCRIPTION
**Pull request by mhegazy**
Fixes #2620

### CODE REVIEW

File: src/compiler/commandLineParser.ts [hunk 1]
```diff
@@ -164,7 +164,12 @@ module ts {
         }
     ];
 
-    /* @internal */
+    /**
+     * Parses a string into CompilerOptions
+     * Note: This assumes a commandline environment, and will attempt to read
+     *       files using ts.sys e.g. response files.
```
Comment (DanielRosenwasser) [inline src/compiler/commandLineParser.ts]: sys.ts

File: src/compiler/commandLineParser.ts [hunk 2]
```diff
@@ -164,7 +164,12 @@ module ts {
         }
     ];
 
-    /* @internal */
+    /**
+     * Parses a string into CompilerOptions
```
Comment (DanielRosenwasser) [inline src/compiler/commandLineParser.ts]: Not true, this parses an array of whitespace-delimited strings.

File: src/compiler/commandLineParser.ts [hunk 3]
```diff
@@ -275,9 +280,9 @@ module ts {
     }
 
     /**
-      * Read tsconfig.json file
-      * @param fileName The path to the config file
-      */
+     * Read tsconfig.json file
+     * @param fileName The path to the config file
+     */
     export function readConfigFile(fileName: string): any {
```
Comment (DanielRosenwasser) [inline src/compiler/commandLineParser.ts]: Why don't we instead make a `parseConfigFile` and expose that?

Comment (DanielRosenwasser) [inline src/compiler/commandLineParser.ts]: Never mind, I just scrolled down. :smile: 

So do we really need to expose this?

File: src/compiler/commandLineParser.ts (hunk 1 above)
Comment (mhegazy) [inline src/compiler/commandLineParser.ts]: i meant the runtime object ts.sys

File: src/compiler/commandLineParser.ts (hunk 3 above)
Comment (mhegazy) [inline src/compiler/commandLineParser.ts]: I am waiting on @clausreinke to confirm. i would rather not expose it though,

File: src/compiler/commandLineParser.ts (hunk 2 above)
Comment (mhegazy) [inline src/compiler/commandLineParser.ts]: will fix it.

**Pull request closed without merging**